### PR TITLE
Updates to index.html and my media q

### PR DESCRIPTION
**Pull request by MindfulOlivka**

### CODE REVIEW

**Review by blissfulmondays — CHANGES_REQUESTED** [review]
Can not merge as the index page has old code in it that needs to be updated with the current main repo files

**Comment by blissfulmondays** [comment]
Please resubmit changes with either up to date repo files or only push your part1 files

File: styles.css [hunk 1]
```diff
@@ -428,45 +428,38 @@ nav {
...
+    transform: translateX(1000px);
   }
-   100% {
+  100% {
     opacity: 0;
     display: none;
-   }
   }
- 
+}
```
Comment (blissfulmondays) [inline styles.css:455]: please remove this - not needed

Comment (MindfulOlivka) [reply styles.css:455]: I don't have it in my last version of css. I still believe } is needed to close key frames.

**Pull request closed without merging**